src/validations/css/stylelint.js

```diff
@@ -0,0 +1,65 @@
+import Validator from '../Validator';
+
+const errorMap = {
+  // CSS Syntax Errors
+  'Unclosed block': (error) => {
```

Comment:
Since we can get either errors or lint violations, I think we should namespace the keys here—e.g. `errors/Unclosed block` or `lint/declaration-block-trailing-semicolon`. Then `keyForError()` can construct the key appropriately.